src/applications/lgy/coe/form/components/SubwayMap.jsx

```diff
@@ -88,7 +88,7 @@ const SubwayMap = () => (
           </ul>
         </va-additional-info>
       </li>
-      <li className="process-step list-three">
+      <li className="process-step list-three vads-u-padding-bottom--3">
```

Comment:
I think you can remove the `process-step` and `list-*` classes from the `li` tags

Reply:
thanks for calling that out, updated!